lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -756,6 +756,15 @@ def test_platform_language_is_used_for_logged_in_user(self):
             self.assertIn("You have been", subject)
             self.assertIn("You have been", message)
 
+    def test_platform_language_is_used_for_new_user(self):
+        # simulate a new user
+        subject, message = self.get_subject_and_message(None)
+        language_after_rendering = get_language()
+
+        self.assertIn("You have been invited in platform language", subject)
+        self.assertIn("You have been invited in platform language", message)
+        self.assertEqual(settings.LANGUAGE_CODE, language_after_rendering)
```

Comment:
I still don't think this is the right test, given the way your pull request description is. This is how you describe the bug you are trying to solve: 
> When the system already been localized that the language code is not "en", it will be a problem because the translation has been deactivated and fallback to use the original strings (in English).

The test you want does the following, I think:

1. Sets the system language to a non-English language
2. Sends a user an email, and asserts that the email appears in a non-English language

Right now you are just testing that the fallback language is English. This test passes without your changes. You need to write a unit test that **fails** before your changes to `lms/djangoapps/instructor/enrollment.py` are applied.